lib/ansible/modules/cloud/amazon/iam_managed_policy.py

```diff
@@ -277,7 +287,7 @@ def main():
         policy=dict(type='json'),
         make_default=dict(type='bool', default=True),
         only_version=dict(type='bool', default=False),
-        fail_on_delete=dict(type='bool', default=True),
+        fail_on_delete=dict(type='bool'),
```

Comment:
```suggestion
        fail_on_delete=dict(type='bool', removed_in_version='2.14'),
```
This needs a changelog fragment and porting guide entry.